packages/sdk/src/oracle/services/statemachines/pollActiveUser.ts

```diff
@@ -0,0 +1,33 @@
+import { Update } from "../update";
+import Store from "../../store";
+import { Handlers as GenericHandlers } from "../../types/statemachine";
+import { ContextClient } from "./utils";
+import { ignoreError } from "../../utils";
+
+export type Params = undefined;
+export type Memory = { error?: Error; iterations: number };
+
+export function initMemory(): Memory {
+  return { iterations: 0 };
+}
+
+export function Handlers(store: Store): GenericHandlers<Params, Memory> {
```

Comment:
Just a high-level comment: do we rely on this poll to update after the user sends the approval transaction, for instance? Or do we force-update after that (is it possible to force-update from the FE)?

The only reason I ask is that a 30s poll can create a bad UX if the user has to wait that long to see the UI update.

Reply:
when we know that the user is interacting with our app, we will update users balance after a transaction is confirmed. This is only for the case where the balance could change external to our app.  This is mainly a problem when testing, as we might seed the account with a bunch of uma through an external script, but it will require a refresh of the app to detect it.

Reply:
[this is an example](https://github.com/UMAprotocol/protocol/blob/master/packages/sdk/src/oracle/services/statemachines/disputePrice.ts#L47) of handling the dispute transaction, and updating the users balance automatically after tx confirms

Reply:
Understood. Makes sense!